packages/app/src/containers/ThemeTextStyleInspector.tsx

```diff
@@ -0,0 +1,236 @@
+import {
+  useApplicationState,
+  useSelector,
+} from '../contexts/ApplicationStateContext';
+import useShallowArray from '../hooks/useShallowArray';
+import FillInspector from './FillInspector';
+import withSeparatorElements from '../utils/withSeparatorElements';
+import NameInspector from '../components/inspector/NameInspector';
+import OpacityInspector from './OpacityInspector';
+import BorderInspector from './BorderInspector';
+import ShadowInspector from './ShadowInspector';
+import { delimitedPath } from 'noya-utils';
+import TextOptionsRow, {
+  SimpleTextDecoration,
+} from '../components/inspector/TextOptionsRow';
+import TextAlignmentRow from '../components/inspector/TextLayoutRow';
+import TextStyleRow from '../components/inspector/TextStyleRow';
+import getMultiValue from '../utils/getMultiValue';
+
+const TextStyleInspector = memo(function TextStyleInspector() {
+  const [, dispatch] = useApplicationState();
+
+  const seletedText = useShallowArray(useSelector(Selectors.getSelectedStyles));
+
+  const getTextStyleAttributes = useCallback((styles: Sketch.Style[]) => {
+    const textStyle = styles[0]?.textStyle;
+    const encodedAttributes = textStyle?.encodedAttributes;
+    const paragraphStyle = encodedAttributes?.paragraphStyle;
+
+    const color = {
+      _class: 'color',
+      red: 0.5,
+      blue: 0.5,
+      green: 0.5,
+      alpha: 0.5,
+    } as Sketch.Color;
+
+    return {
+      fontColor: encodedAttributes?.MSAttributedStringColorAttribute || color,
+      fontFamily:
+        encodedAttributes?.MSAttributedStringFontAttribute.attributes.name ||
+        'Arial',
+      fontSize: getMultiValue<number | undefined>(
+        styles.map(
+          (s) =>
+            s.textStyle?.encodedAttributes.MSAttributedStringFontAttribute
+              .attributes.size,
+        ),
+      )
+        ? encodedAttributes?.MSAttributedStringFontAttribute.attributes.size
+        : undefined,
+      lineHeight: getMultiValue<number | undefined>(
+        styles.map(
+          (s) =>
+            s?.textStyle?.encodedAttributes.paragraphStyle?.maximumLineHeight,
+        ),
+      )
+        ? paragraphStyle?.maximumLineHeight ?? 22
+        : undefined,
+      horizontalAlignment: getMultiValue<
+        Sketch.TextHorizontalAlignment | undefined
+      >(
+        styles.map(
+          (s) => s?.textStyle?.encodedAttributes.paragraphStyle?.alignment,
+        ),
+      )
+        ? paragraphStyle?.alignment ?? Sketch.TextHorizontalAlignment.Left
+        : undefined,
+      textTransform: getMultiValue<Sketch.TextTransform | undefined>(
+        styles.map(
+          (s) =>
+            s?.textStyle?.encodedAttributes
+              ?.MSAttributedStringTextTransformAttribute,
+        ),
+      )
+        ? encodedAttributes?.MSAttributedStringTextTransformAttribute ??
+          Sketch.TextTransform.None
+        : undefined,
+      textDecoration: encodedAttributes?.underlineStyle
+        ? 'underline'
```

Comment:
If we do `'underline' as const`, `'strikethrough' as const`, ...,  we should be able to avoid the `textDecoration as SimpleTextDecoration` below